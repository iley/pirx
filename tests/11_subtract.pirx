func main() {
  var x: int;
  var y: int;
  x = 10;
  y = 3;
  printf("x - y = %d\n", x - y);
  printf("x - 1 = %d\n", x - 1);
  printf("7 - y = %d\n", 7 - y);
  printf("x + y - 1 = %d\n", x + y - 1);
  return 0;
}
