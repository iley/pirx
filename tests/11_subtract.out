x - y = 7
x - 1 = 9
7 - y = 4
x + y - 1 = 12
